### PR TITLE
Pr94x l1t Muon Unpacker Zero Suppression and DQM. Clang cleaning. 

### PR DESCRIPTION
**Pull request by rekovic**
PR 94x - Upgrade the Muon Unpacker and DQM to inteligentely handle Zero Suppression, making it BX aware.   Also includes CLANG cleaning 

Needed for data taking and available in the release at P5 before hardware change goes online (soon).
- This is compatible with past, current, and future  data.

### CODE REVIEW

**Comment by rekovic** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
+1

**Comment by rekovic** [comment]
@dmitrijus Please sign/comment.

**Comment by dmitrijus** [comment]
Please remove changes to:
DQM/Integration/python/config/*

Otherwise it is fine and can go in.

**Comment by dmitrijus** [comment]
Ah! DQM/Integration/python/config/FrontierCondition_GT_cfi.py still remains.

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
+1

**Comment by rekovic** [comment]
@dmitrijus done.

**Comment by dmitrijus** [comment]
+1

File: EventFilter/L1TRawToDigi/interface/BxBlock.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#ifndef EventFilter_L1TRawToDigi_BxBlock_h
```
Comment (davidlange6) [inline EventFilter/L1TRawToDigi/interface/BxBlock.h]: hi @rekovic - this class is a data format - please move it to a DataFormats package

Comment (davidlange6) [reply EventFilter/L1TRawToDigi/interface/BxBlock.h]: (as are other classes in this package, but lets at least not add new ones)

**Comment by rekovic** [comment]
will rebase

**Pull request closed without merging**